### DISCUSSION TITLE
Support for extended ASCII characters

### DISCUSSION THREAD

**Discussion by vijayprakash1** [olikraus/u8g2#2234] [Q&A]
Hi, 

Is there support for extended ASCII characters, I want to display languages like Spanish, French, German etc. That has Latin words.

I am new to this so wanted to check if is there any solution for the same or guidance on how I can develop the library and logic for extended characters.

**Comment by olikraus** [comment]
If the fontname ends with "e", then it may contain the extended unicodes for the mentioned languages, see: https://github.com/olikraus/u8g2/wiki/u8g2reference#setfont

**Reply by vijayprakash1** [reply]
Hi @olikraus ,

Is there extended ASCII character library in hex format? Or please guide me how to generate these custom fonts library

**Reply by olikraus** [reply]
U8g2 is based on Unicode (no ASCII support as such). Custom fonts must be available in BDF file format. 
A lot of questions are answered in the U8g2 FAQ: 
https://github.com/olikraus/u8g2/blob/eb00ddd9f464f0a6e05853d65dfba927a36d1ea8/doc/faq.txt#L255-L291
